### ISSUE TITLE
Testen/Implementieren weiterer Optimizer, Losses

### ISSUE THREAD

**Issue by jerome-rutinowski** [psorus/eereid#3]
SGD, Cross Entropy als Beispiel implementieren (ggf. nur weiterer Import nötig)

**Comment by psorus** [comment]
SGD kann momentan mit der variable "optimizer" gesetzt werden
g["optimizer"]="sgd" sollte funktionieren

**Comment by psorus** [comment]
Cross Entropy ist eine gute Idee. Gerne Bescheid sagen wenn meine loss files nicht verständlich sind

**Comment by jerome-rutinowski** [comment]
> SGD kann momentan mit der variable "optimizer" gesetzt werden g["optimizer"]="sgd" sollte funktionieren

Wollen wir am Ende nicht eher sowas haben: g=ee.ghost(optimizer=tf.keras.optimizers.SGD)?

**Comment by psorus** [comment]
witzigerweise funktioniert das auch

keras unterstützt strings oder optimizer, und g["variable"]=something kann man auch als ghost(variable=something) schreiben

**Comment by jerome-rutinowski** [comment]
Ok, dann hab ich irgendwas falsch gemacht! @fpolachowski, wenn das bei dir auch einfach klappt gerne kurz die richtige Syntax hier reinpacken und issue schließen.
